package.json

```diff
@@ -3,97 +3,73 @@
-    "watch": "webpack --progress --colors --watch --mode development",
-    "release": "standard-version",
+    "precommit": "npm run build:prod && npm run type-check && npm run lint",
     "pushTaggedRelease": "git push --follow-tags --no-verify origin master",
-    "eslint": "eslint . --color",
-    "flow": "flow check",
-    "commit:dist": "git add --force --all dist && (git commit -m 'chore: update dist' || exit 0)",
-    "precommit": "lint-staged",
-    "docs:generate": "documentation build flow-typed/** src/** -f md -o docs/configuration-api.md",
-    "docs:serve": "documentation serve flow-typed/** src/** --watch",
-    "prettier:fix": "prettier --write ."
-  },
-  "lint-staged": {
-    "*.{js,jsx}": [
-      "eslint --fix",
-      "git add"
-    ],
-    "*.{json,md,scss}": [
-      "prettier --write",
-      "git add"
-    ]
+    "release": "standard-version"
   },
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
-    "@babel/cli": "^7.10.5",
-    "@babel/core": "^7.10.5",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
-    "@babel/preset-env": "^7.10.4",
-    "@babel/preset-flow": "^7.10.4",
-    "@babel/register": "^7.10.5",
-    "@playkit-js/info": "3.4.0-canary.0-c282105",
-    "@playkit-js/kaltura-player-js": "latest",
-    "@playkit-js/moderation": "3.2.13-canary.0-5376147",
-    "@playkit-js/playkit-js": "latest",
-    "@playkit-js/playkit-js-downloads": "1.1.0-canary.0-6acbd10",
-    "@playkit-js/playkit-js-providers": "2.39.4-canary.0-ab0b9a0",
-    "@playkit-js/related": "latest",
-    "@playkit-js/share": "https://github.com/kaltura/playkit-js-share.git#726fd2d383ee04c2889ec76ff390581998b339bb",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-istanbul": "^6.0.0",
-    "chai": "^4.2.0",
-    "conventional-github-releaser": "3.1.3",
-    "copy-webpack-plugin": "^6.0.3",
-    "cross-env": "^7.0.2",
-    "eslint": "^7.4.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-loader": "^4.0.2",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-mocha-no-only": "^1.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "flow-bin": "^0.129.0",
-    "husky": "^4.2.5",
-    "istanbul": "^0.4.5",
-    "karma": "^5.1.0",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-cli": "^2.0.0",
-    "karma-coverage": "^2.0.2",
-    "karma-firefox-launcher": "^1.3.0",
-    "karma-ie-launcher": "^1.0.0",
+    "@babel/core": "^7.23.7",
+    "@babel/plugin-transform-runtime": "^7.23.7",
+    "@babel/preset-env": "^7.23.8",
+    "@babel/preset-typescript": "^7.23.3",
+    "@babel/runtime": "^7.23.8",
+    "@microsoft/api-extractor": "^7.39.1",
+    "@playkit-js/browserslist-config": "1.0.8",
+    "@playkit-js/kaltura-player-js": "^3.17.10",
+    "@playkit-js/moderation": "canary",
```

Comment:
use explicit versions

Reply:
it is anyway not in use right now and not imported into src temporarily